ow_dynamic_terrain/src/TerrainModifier.cpp

```diff
@@ -250,9 +252,25 @@ bool TerrainModifier::applyImageToHeightmap(Heightmap* heightmap, const cv::Poin
 
   auto diff = OpenCV_Util::createZerosMatLike(image);
 
+  uint32 mask_size = 0;
+  HardwarePixelBufferSharedPtr pixel_buffer;
+  uint8* mask_pixels = nullptr;
```

Comment:
Apparently `uint*` is provided by some compilers, but `uint*_t` is in the standard library. I think we should use the standard library versions. This comment applies to all instances of these types that in other parts of the code.
```suggestion
  uint32_t mask_size = 0;
  HardwarePixelBufferSharedPtr pixel_buffer;
  uint8_t* mask_pixels = nullptr;
```

We could also leave this for a refactor.

Reply:
Fixed